crates/trie/parallel/Cargo.toml

```diff
@@ -58,12 +55,11 @@ proptest.workspace = true
 proptest-arbitrary-interop.workspace = true
 
 [features]
-default = ["metrics", "async", "parallel"]
+default = ["metrics", "parallel"]
 metrics = ["reth-metrics", "dep:metrics", "reth-trie/metrics"]
-async = ["tokio/sync", "itertools"]
-parallel = ["rayon"]
+parallel = ["rayon", "itertools"]
```

Comment:
let's get rid of this feature, since this is the only impl that crate exposes

Reply:
sure done here https://github.com/paradigmxyz/reth/pull/11213/commits/456db3b088220e728c2e801552f13d58797341ca